docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
 
 ## Contributing
 
-Feel free to test different processors and report your results via
-[email](mailto:contact@dasharo.com) or submit a Pull Request to
-[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
-[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Use [Dasharo Tools Suite HCL
+  report](../../../dasharo-tools-suite/documentation/#hcl-report) to upload
+  report automatically.
+* Create new issue in [Dasharo issues
```

Comment:
We can also suggest issue title by adding `&title=` to the URL, similarly as with e-mail subject

Reply:
We can only address some of those suggestions, and we should decide what would be best.

The link was improved here: https://github.com/Dasharo/docs/pull/463/commits/3b09c4e70a6d6c134d5c2307b3870fd7adafddbb.

I used the same title for the email, but I disagree with the whole concept of HCL reporting over GH. If we had separate small HCL reports for CPU/GPU/Memory, it would work, but I opt for minimizing user effort and maximally automating the work of HCL generation through CI.